packages/main/src/TimePickerPopoverTemplate.tsx

```diff
@@ -71,3 +76,31 @@ export default function TimePickerPopoverTemplate(this: TimePicker) {
 		</>
 	);
 }
+
+function valueStateMessage(this: TimePicker) {
+	return (
+		<>
```

Comment:
You don't need the <> and { on the next line, you can direclty return the JSX expression